apps/storefront/src/shared/service/b2b/graphql/users.ts

```diff
@@ -177,9 +177,23 @@ export interface UsersResponse {
   };
 }
 
-export const getUsers = (data: CustomFieldItems) =>
-  B3Request.graphqlB2B({
-    query: getUsersQl(data),
+export interface GetUsersVariables {
+  first: number;
+  offset: number;
+  q?: string;
+  companyId: number | string;
+  companyRoleId?: number | string;
+}
```

Comment:
horribly vague interface, but the aim for this PR is just to document what we currently have